src/Core/Grid/Definition/Factory/AddressGridDefinitionFactory.php

```diff
@@ -290,6 +291,11 @@ protected function getBulkActions(): BulkActionCollectionInterface
                     ->setOptions([
                         'submit_route' => 'admin_addresses_delete_bulk',
                         'confirm_message' => $this->trans('Delete selected items?', [], 'Admin.Notifications.Warning'),
```

Comment:
Should we keep _items_? Do you have a kind of code template for modal? It would be nice to add a wording template as well so that all modal can look the same, work the same and 'talk' the same. :-)

Reply:
good idea, I'll see what I can do with other similar translations @LouiseBonnard

Reply:
```suggestion
                        'confirm_message' => $this->trans('Are you sure you want to delete the selected address(es)?', [], 'Admin.Notifications.Warning'),
```